doc/man7/provider-digest.pod

```diff
@@ -152,11 +154,16 @@ Any parameter settings are additional to any that were previously set.
 OSSL_FUNC_digest_get_ctx_params() gets digest operation details details from
 the given provider side digest context I<dctx> and stores them in I<params>.
 
-OSSL_FUNC_digest_gettable_params(), OSSL_FUNC_digest_gettable_ctx_params(), and
-OSSL_FUNC_digest_settable_ctx_params() all return constant B<OSSL_PARAM> arrays
-as descriptors of the parameters that OSSL_FUNC_digest_get_params(),
-OSSL_FUNC_digest_get_ctx_params(), and OSSL_FUNC_digest_set_ctx_params() can handle,
-respectively.
+OSSL_FUNC_digest_gettable_params() returns a constant B<OSSL_PARAM> array
+containing desriptors of the parameters that OSSL_FUNC_digest_get_params()
+can handle.
+
+OSSL_FUNC_digest_gettable_ctx_params() and
+OSSL_FUNC_digest_settable_ctx_params() both return constant
+B<OSSL_PARAM> arrays as descriptors of the parameters that
+OSSL_FUNC_digest_get_ctx_params() and OSSL_FUNC_digest_set_ctx_params()
+can handle, respectively.  The array is based on the current state of the
+context if I<dctx> is not NULL and on the algorithm I<provctx> otherwise.
```

Comment:
digest side provider context
algorithm ??

Reply:
Poor choice of naming.  `provctx` refers to the algorithm returned by the provider to libcrypto.  @levitte [noted this above](https://github.com/openssl/openssl/pull/14240#discussion_r578879244).

I'm not proposing to change this one globally.

Reply:
Agreed, let's make that in another PR.  The MAC API uses `data` instead, which turns out to be much less confusing.

Reply:
+1 for `data`

Reply:
Someone raise the issue?

Reply:
Issue #14284 raised.